app_dart/lib/src/request_handlers/scheduler/vacuum_stale_tasks.dart

```diff
@@ -53,32 +53,16 @@ class VacuumStaleTasks extends RequestHandler<Body> {
     final DatastoreService datastore = datastoreProvider(config.db);
 
     final List<FullTask> tasks = await datastore.queryRecentTasks(slug: slug).toList();
-    final Set<Task> tasksToBeReset = <Task>{};
+    final List<Task> tasksToBeReset = <Task>[];
```

Comment:
Is this to allow duplicated entries?

Reply:
We should not have duplicated entries. The query above returns recent tasks and all are different entries.